java-library/src/main/java/com/microsoft/azure/functions/sql/annotation/SQLInput.java

```diff
@@ -17,6 +17,11 @@
 @Target(ElementType.PARAMETER)
 @CustomBinding(direction = "in", name = "sql", type = "sql")
 public @interface SQLInput {
+    /**
+     * The variable name used in function.json.
```

Comment:
This isn't very clear to me - maybe something more like 

```suggestion
     * The unique name to give the binding in the function.json
```

Reply:
Wanted to keep the comments the same as other binding libraries to avoid confusion for users working with multiple bindings:
https://github.com/Azure/azure-functions-java-library/blob/5fe66b0dbf0e6973abfd618ac100813e8ac48bde/src/main/java/com/microsoft/azure/functions/annotation/CosmosDBInput.java#L57